lib/tictactoe.rb

```diff
@@ -24,8 +26,8 @@ def show_board
   end
 
   def play_turn
-    position = @player.selection
-    @board.place_token(position, "X")
+    position = @player.selection(@board)
```

Comment:
I like to use dedicated attributes readers (and writes is necessary) for instance variables. One of the benefits is you get a better signal if something goes wrong.

For instance, if you had written this, `@boarb` will evaluate to `nil`
```suggestion
    position = @player.selection(@boarb)
```

But with an `attr_reader`

```suggestion
    position = @player.selection(boarb)
```
This will result in a `NoMethodError` and will not pass a bad value to `player.selection`

Reply:
will adjust throughout all classes

Reply:
I did for all classes except the board class.